docusaurus/docs/get-started/get-started.mdx

```diff
@@ -135,82 +167,52 @@ Depending on the answers you gave to the prompts, there should now be a structur
 └── tsconfig.json
 ```
 
-When you've finished installing the tools, open the plugin folder:
-
-<CodeSnippets
-  snippets={[
-    { component: InstallNPM, label: 'npm' },
-    { component: InstallPNPM, label: 'pnpm' },
-    { component: InstallYarn, label: 'yarn' },
-  ]}
-  groupId="package-manager"
-  queryString="current-package-manager"
-/>
-
-## Create-plugin prompts reference
-
-When running the `create-plugin` command, the following prompts appear:
+For more information about these files, refer to [Folder structure](get-started/folder-structure/).
 
-### What is the name of your plugin?
+### Step 3: Run your plugin in Docker
 
-Enter the name of your plugin. This helps to identify its purpose.
+With the `create-plugin` tool, you can use a Docker container to simplify the configuration, loading, and development processes. For more information, refer to [Set up development environment](get-started/set-up-development-environment/).
 
-### What is the organization name of your plugin?
+### Step 4: Use CLI commands
```

Comment:
There's a fair bit of overlap between steps 3 and 4. I'd be inclined to remove step 4 and just detail the key commands in step 3:
- install (npm i)
- build the frontend in development mode (npm run dev)
- build the backend (optional) (mage -v build:linux)
- run the Grafana server (npm run server)

Reply:
done.